src/hmi.c

```diff
@@ -220,3 +220,33 @@ uint8_t speed_mode_configuration(void)
 		}
 	}
 }
+
+/**
+ * @brief Function to choose if reuse the saved eeprom maze.
+ *
+ * @return true if the maze saved on EEPROM is restored to RAM
+ */
+bool reuse_maze(void)
+{
+	if (eeprom_maze_is_saved()) {
+		led_bluepill_on();
+		LOG_INFO("maze saved");
+		while (1) {
+			if (button_right_read_consecutive(500)) {
+				eeprom_load_maze();
+				LOG_INFO("right button");
+				return true;
+			}
+			if (button_left_read_consecutive(500)) {
+				LOG_INFO("left button");
+				eeprom_reset_maze();
+				led_bluepill_off();
+				return false;
+			}
+		}
+	} else {
+		led_bluepill_off();
+		LOG_INFO("maze no saved");
+		return false;
+	}
```

Comment:
Just to be a bit more consistent with the current HMI, would you mind loading when left button is pressed and resting when right button is pressed?

When selecting speed, for example, we change configuration with right button and "confirm" with left. In this case instead of confirming would be an "ok/yes" for loading. What do you think?

Reply:
ok